.github/workflows/dev-stg-release.yml

```
@@ -24,6 +24,11 @@ jobs:
         with:
           java-version: 11
 
+      - name: Set Up Ballerina
+        uses: ballerina-platform/setup-ballerina@v1.1.0
+        with:
+          version: latest
```

For the `release` and `dev-stg-release` workflows, we have to specify the exact ballerina version (which should be 2201.7.0), to avoid releasing connectors with unexpected ballerina versions ( as `version: latest` will always pick the latest available version)

```suggestion
      - name: Set Up Ballerina
        uses: ballerina-platform/setup-ballerina@v1.1.0
        with:
          version: 2201.7.0
```